node/src/cli.rs

```diff
@@ -93,7 +112,7 @@ pub struct ProviderConfigurations {
     pub storage_layer: Option<StorageLayer>,
 
     /// Storage location in the file system
-    #[clap(long, required_if_eq("storage-layer", "rocks-db"))]
+    #[clap(long, required_if_eq("storage_layer", "rocks-db"))]
```

Comment:
Most arguments in a substrate node follow the `this-casing`, why using a different kind?

Reply:
This was changed due to this error:
```
Command storage-hub-node: Argument or group 'storage-layer' specified in 'required_if_eq*' for 'storage_path' does not exist
```

Reply:
My bad, I misinterpreted some stuff. All good.